zookeeper-server/src/main/java/org/apache/zookeeper/common/X509Util.java

```diff
@@ -238,18 +239,19 @@ public int getSslHandshakeTimeoutMillis() {
         return result;
     }
 
+    @SuppressWarnings("unchecked")
     public SSLContext createSSLContext(ZKConfig config) throws SSLContextException {
-        if (config.getProperty(sslClientContextProperty) != null) {
-            LOG.debug("Loading SSLContext from property '" + sslClientContextProperty + "'");
-            String sslClientContextClass = config.getProperty(sslClientContextProperty);
+        if (config.getProperty(sslContextSupplierClassProperty) != null) {
+            LOG.debug("Loading SSLContext supplier from property '" + sslContextSupplierClassProperty + "'");
```

Comment:
nit: enclose this in a `if (LOG.isDebugEnabled()) { ... }`

Reply:
Also for readability, consider using `{}` formatting to build the log message:

`LOG.debug("Loading SSLContext supplier from property '{}'},  sslContextSupplierClassProperty);`

Reply:
Yep, no problem 👍